### PR TITLE
Reliability Analysis: Bug fix, Error handling and added Average inter item correlation.

### PR DESCRIPTION
**Pull request by vandenman**
Reliability Analysis: Bug fix, Error handling and added Average inter item correlation.

### CODE REVIEW

**Comment by boutinb** [comment]
@vandenman Moet je die fix herschreven met die nieuwe error handling van Tim?

File: JASP-Engine/JASP/R/reliabilityanalysis.R [hunk 1]
```diff
@@ -16,24 +16,26 @@
 #
 
 ReliabilityAnalysis <- function(dataset = NULL, options, perform = "run",
-						   callback = function(...) 0,  ...) {
-
+								callback = function(...) 0,  ...) {
+
```
Comment (akashrajkn) [inline JASP-Engine/JASP/R/reliabilityanalysis.R]: Hi @vandenman, can you please remove the trailing spaces and push the code again? (It's difficult to see the diff)

**Comment by vandenman** [comment]
there appear to be some inconsistencies in the use of whitespace in the original file (see for example line 50 in the old file which has 6 tabs while the code starts on 3 tabs). I now changed the standard to 0 whitespace on empty lines. Let me know if I missed anything. The files might be a bit confusing now, but you can add "?w=1" to the end of an url to show changes without whitespace (as explained [here](https://github.com/tiimgreen/github-cheat-sheet#ignore-whitespace)).

**Comment by boutinb** [comment]
@vandenman: For the next time, if you correct layout or white space, can you do that in a apart commit? That would make much easier to see the real changes (the "?w=1" trick is not always known, and will be forgotten...).

**Pull request merged**